src/BuildIt.Forms/BuildIt.Forms.Controls.Android/Resources/Resource.Designer.cs

```diff
@@ -2187,9 +2187,27 @@ public partial class Drawable
 			// aapt resource value: 0x7f020053
```

Comment:
Is there a reason why this file was committed?

Reply:
ah no, didn't realised it picked up on that change weird